docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Codacy scans your code for issues and calculates the metrics code complexity, duplication, and coverage. Besides this, Codacy also calculates a grade for your repository and files based on all calculated code quality metrics.
+---
+
+# Which metrics does Codacy calculate?
+
+Codacy performs static code analysis and calculates code duplication, code complexity, and code coverage metrics for [most supported programming languages](../../getting-started/supported-languages-and-tools.md).
+
+The following sections describe how Codacy calculates each supported metric and where you can see each metric on the Codacy UI:
+
+-   [Grade](#grade)
+-   [Issues](#issues)
+-   [Complexity](#complexity)
+-   [Duplication](#duplication)
+-   [Code coverage](#code-coverage)
+
+!!! note
+    Depending on certain characteristics of your repository, such as the number of source code files and their number, Codacy may [apply limits to the code analysis](does-codacy-place-limits-on-the-code-analysis.md) that impact the calculation of the supported metrics.
```

Comment:
@prcr Is it possible this is a repetition error here? "such as the number of source code files and their number"

Reply:
@prcr Also, missing and "s" in impactS - "that impact the calculation"

Reply:
> Is it possible this is a repetition error here? "such as the number of source code files and their number"

Indeed, I meant "[...] such as the number of source code files and their **size**"! :+1: 

>  Also, missing and "s" in impactS - "that impact the calculation"

Here the verb "impact" must agree in number with the plural subject "limits", so it's actually correct: "the limits impact the calculation".